Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -37,18 +37,23 @@ public struct ${type.buildable()}${conformance_clause(conformances)} {
   let ${child.name()}: ${child.type().buildable()}
 %   end
 
+  /// The leading trivia attached to this syntax once built.
```

Comment:
I got a little confused what this `leadingTrivia` and the `additionalLeadingTrivia` were used for. What do you think about adding a comment like
```swift
/// This is typically used to add comments to syntax nodes
```

to this member and something like

```swift
/// `leadingTrivia` is typically used to add the correct amount of indentation to this node.
```
to the `build*(` method.